packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1818,11 +1818,33 @@ class BackdropFilterLayer extends ContainerLayer {
     }
   }
 
+  /// The blend mode to use to apply the filtered background content onto the background
+  /// surface.
+  ///
+  /// The default mode is [BlendMode.srcOver] which is the most compatible mode, but
+  /// using this widget inside of a parent that uses a saveLayer may produce surprising
+  /// results. When rendering inside a saveLayer which implicitly presents a transparent
+  /// background, the results would look better with a [BlendMode.src] mode. Note that
+  /// the DOM-html renderer on web does not support this parameter so using any mode
+  /// but the default may produce different results.
+  ///
+  /// The scene must be explicitly recomposited after this property is changed
+  /// (as described at [Layer]).
+  BlendMode? get blendMode => _blendMode;
+  BlendMode? _blendMode;
```

Comment:
The docs above says the default is "BlendMode.srcOver", but here it doesn't seem to have a default value?

Reply:
Yes, because the layers don't follow the pattern used elsewhere. Most other objects in the chain are constructed with a value that has a default and then can have their property updated afterwards, but layers are constructed with no arguments and then have parameters set manually - thus they have no way to enforce defaults.

This is not a place to use a template/macro as a result.

Reply:
Switched to defaulting it when the value is passed on.